src/giph

```diff
@@ -49,6 +68,7 @@ OPTIONS
   -ms, --microphone-source=STRING   Overwrite the default microphone source.
   -y,  --notify                     Send notification on error or success.
        --stop                       Finish any running giph recordings.
+       --finish                     Finish any screen recordings only.
```

Comment:
The intention of `--stop` is to stop all recordings. So I would like your implementation to replace it. Optionally, you could add a `--kill` option to just abort all recordings.

Reply:
You're right. I will change relative codes and make these flag names match their functionsa.